flow/tutorials/scripts/drt/drc_fix.tcl

```diff
@@ -5,32 +5,28 @@ read_lef ../../../platformms/sky130hd/lef/sky130_fd_sc_hd_merged.lef
 read_def ./gcd/4_cts.def
```

Comment:
update `platformms` to `platforms` in line 4

Reply:
Fixed on [556ae52](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/527/commits/556ae5268c74eb3f7c40fc2f34723f36fc971644)